main/acle.md

```diff
@@ -6816,19 +6816,29 @@ operation that they perform.
 
 #### List of instructions
 
-This section contains a list of all FEAT_SVE instructions. For each one it
+This section contains a list of all SVE instructions. For each one it
```

Comment:
Hi @rsandifo-arm - would it make sense to group the instructions by `__ARM_FEATURE_*`?

Reply:
I guess both ways lose something.  The current way doesn't separate instructions from different ISA features, whereas grouping them based on ISA feature would lose the simple alphabetical list.

FWIW, here are some arguments in favour of keeping it as it is:

* It matches the style used in the [ISA documentation](https://developer.arm.com/documentation/ddi0602/2022-03/SVE-Instructions/), which makes it easier to do a 1:1 comparison.

* The difference between SVE and SVE2 is likely to become less important over time, at least for new authors, so having them in a single list should gradually become more useful than having separate lists.

* The requirements for some instructions, such as the F64 matmul instructions, are more complicated than a single FEAT, so even grouping by `__ARM_FEATURE_*` will miss some information.  The current approach ensures that any information about availability is provided by the ISA documentation instead (at the cost of two hops: one from the ACLE to the intrinsics, and one from the intrinsic documentation to the instruction documentation).

Also, I don't think this list is a good starting point for exploring the ISA.  The interactive intrinsics themselves are probably better for that.  I'd imagined that this list would be more useful for people who already know which instruction they want to use and want to find the associated intrinsic.

Reply:
Thank you for the clear explanation! I agree with you.